api/src/main/java/com/codeforcommunity/rest/FailureHandler.java

```diff
@@ -210,6 +192,18 @@ public void handleMemberStatusException(RoutingContext ctx, MemberStatusExceptio
     end(ctx, message, 400);
   }
 
+  public void handleWrongFavoriteStatusException(RoutingContext ctx, WrongFavoriteStatusException e) {
+    String message = "";
```

Comment:
You don't have to initialize this variable as an empty string, just leave it as a declaration: `String message;`

Reply:
Didn't know that, thanks!